bolognese.js

```diff
@@ -0,0 +1,56 @@
+import {spaghetti, passata, beef, onion, celery, carrot, garlic} from "store";
+import {salt, pepper, oliveOil, water, saucepan, pot} from "kitchen";
+import {basil, oregano} from "spices";
+
+onion.take(1, 'piece').dice();
+celery.take(150, 'g').dice();
+carrot.take(150, 'g').dice();
+garlic.take(2, 'clove').chop();
+beef.take(400, 'g').mince();
+
+saucepan.add(oliveOil, 1, 'tbsp');
+saucepan.add(beef);
+saucepan.add(salt, 1, 'pinch');
+saucepan.add(pepper, 2, 'pinch');
+
+while(!beef.browned()){
+    saucepan.heat('medium');
```

Comment:
ja bi prvo ugrijao tavu, ako stavis na hladnu tavu meso ce pustiti vodu, što nije ok, isto tako povecao bi ulje i stavio jacu vatru, barem u početku